x-pack/plugins/transform/public/app/sections/transform_management/components/transform_list/transforms_stats_bar.tsx

```diff
@@ -87,12 +100,50 @@ function createTranformStats(transformsList: TransformListRow[]) {
   return transformStats;
 }
 
-interface Props {
+interface TransformStatsBarProps {
+  transformNodes: number;
   transformsList: TransformListRow[];
 }
 
-export const TransformStatsBar: FC<Props> = ({ transformsList }) => {
-  const transformStats: TransformStatsBarStats = createTranformStats(transformsList);
+export const TransformStatsBar: FC<TransformStatsBarProps> = ({
+  transformNodes,
+  transformsList,
+}) => {
+  const { esNodeRoles } = useDocumentationLinks();
+
+  const transformStats: TransformStatsBarStats = createTranformStats(
+    transformNodes,
+    transformsList
+  );
 
-  return <StatsBar stats={transformStats} dataTestSub={'transformStatsBar'} />;
+  return (
+    <>
+      <StatsBar stats={transformStats} dataTestSub={'transformStatsBar'} />
+      {transformNodes === 0 && (
```

Comment:
If the node count is zero, would also be nice to disable the Create, Clone and Start actions in the UI, similar to the behavior for ML jobs. 

![image](https://user-images.githubusercontent.com/7405507/110624241-d8a90d80-8195-11eb-89b6-a90766944f40.png)

Could be done in a follow-up, as we do show a clear error message, although the user only gets to see this right at the end of the wizard.

![image](https://user-images.githubusercontent.com/7405507/110624597-553bec00-8196-11eb-9d39-65c018421a12.png)

Reply:
Can you disable the Create button on the management page too?

![image](https://user-images.githubusercontent.com/7405507/110796195-97852c00-826f-11eb-8016-986ce08205e1.png)

Reply:
Pushed an update in e970178 that disables the create button when no nodes are available.